packages/zoe/src/contractSupport/prepare-ownable.js

```diff
@@ -0,0 +1,81 @@
+import { M } from '@endo/patterns';
+import { prepareExoClass, prepareRevocableKit } from '@agoric/vat-data';
+import { OfferHandlerI } from '../typeGuards.js';
+
+const TransferProposalShape = M.splitRecord({
+  give: {},
+  want: {},
+  exit: {
+    onDemand: null,
+  },
+});
+
+export const prepareOwnable = (
+  zcf,
+  baggage,
+  uKindName,
+  uInterfaceName,
+  uMethodNames,
+) => {
+  const makeRevocableKit = prepareRevocableKit(
+    baggage,
+    uKindName,
+    uInterfaceName,
+    uMethodNames,
+    {
+      makeTransferInvitation: M.call().returns(M.promise()),
+    },
+    {
+      makeTransferInvitation() {
+        const {
+          state: {
+            // @ts-expect-error `this` typing
+            underlying,
+          },
+          facets: {
+            // @ts-expect-error `this` typing
+            revoker,
+          },
+        } = this;
+        const customDetails = underlying.getCustomDetails();
```

Comment:
I still think customDetails doesn't say enough. Do you mind `inviteDetails`? Do you have a different theory?

Reply:
I agree with this sentiment. But
- `inviteDetails` is not grammatical because `invite` is a verb, suggesting `invitationDetails`
- But `invitationDetails` already has an existing meaning in our system. See https://github.com/Agoric/agoric-sdk/blob/87e4ef498bd57ac1577e46ecdcc3bb96c0a98c88/packages/zoe/src/zoeService/invitationQueries.js#L8 and https://github.com/Agoric/agoric-sdk/blob/87e4ef498bd57ac1577e46ecdcc3bb96c0a98c88/packages/zoe/src/zoeService/types.js#L41 . 
- The invitation's `customDetails` is a subrecord of an invitation's invitation-details record.

In light of all this, I went with `getInvitationCustomDetails`. I normally hate names this long, but since it needs to be understood as distinct from the object's "normal" method names, the length isn't terrible.

Done.

Reply:
A dissenting voice on whether "invite" can be used as a noun: https://grammarist.com/usage/invite/

Nevertheless, I'm gonna stick with `getInvitationCustomDetails` because it appears on the ownable alongside `makeTransferInvitation`, and I want to suggest that these are related.